### PR TITLE
qt5: upgrade to v5.15.13-lts-lgpl

### PR DESCRIPTION
**Pull request by fmntf**
* SRCREVs updated to 5.15.13-lgpl
* Removed patch on qtbase that has been integrated upstream

Packages successfully built against Yocto Hardknott: qtbase, qtmultimedia, qtwayland. Other recipes are untested for unavailability of machine build time.

### CODE REVIEW

File: recipes-qt/qt5/qtremoteobjects_git.bb [hunk 1]
```diff
@@ -25,6 +25,6 @@ PACKAGECONFIG[tools-only] = "CONFIG+=tools-only"
 
 EXTRA_QMAKEVARS_PRE += "${PACKAGECONFIG_CONFARGS}"
 
-SRCREV = "f64e34be9ac4b7e92c63e47235c04471a1d40c93"
+SRCREV = "29394e35db43acb20e7b4d5e978a733f5b4232a6"
```
Comment (shr-project) [inline recipes-qt/qt5/qtremoteobjects_git.bb:28]: This doesn't seem correct

It's copy paste from:
qtquicktimeline_git.bb:SRCREV = "29394e35db43acb20e7b4d5e978a733f5b4232a6"

will fix that.

Comment (fmntf) [reply recipes-qt/qt5/qtremoteobjects_git.bb:28]: you are right, correct hash is 18ef1cdce7bc4c93415f38f1c220ab697aa75908

Comment (shr-project) [reply recipes-qt/qt5/qtremoteobjects_git.bb:28]: I've fixed it to sha of the tag itself 1d8e4d3645c437cb0a5e5cdb4862949190f89259 in master-next.

**Comment by shr-project** [comment]
Merged, thanks.

**Pull request closed without merging**